nextflow.config

```diff
@@ -54,7 +54,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes'
+    validationSchemaIgnoreParams     = 'genomes,fasta'
```

Comment:
Does fasta really need to be added here? From what I understand [here](https://nextflow-io.github.io/nf-validation/parameters/validation/#ignoring-unrecognized-parameters) `genomes` is set here because is a nested parameter but not `fasta` or maybe I am not understanding it correctly?